programs/programs/policyengine/calculators/dependencies/member.py

```diff
@@ -100,6 +100,42 @@ def value(self):
         return self.member.disabled or self.member.long_term_disability
 
 
+class IsUSDADisabledDependency(Member):
+    field = "is_usda_disabled"
+
+    def value(self):
+        return self.member.has_disability()
+
+
+class IsUSDAElderlyDependency(Member):
+    field = "is_usda_elderly"
+
+    def value(self):
+        return self.member.is_elderly()
```

Comment:
Did you have trouble with these? They should be calculated from the age and disabled parameters we send to PE. We might need to add those as dependencies to SNAP. We typically like to give PE the most root variable, so if changes are made to the rules on PE's side, we don't have to also make a change on our side.

Reply:
Yeah, you are right. Will refactor this out.